addons/spectrum/XEH_preInit.sqf

```diff
@@ -192,4 +192,131 @@ GVAR(voiceLinePacks) set ["british", 			[GVAR(voiceLinesBritishList), GVAR(voice
 GVAR(voiceLinePacks) set ["morsecode", 			[GVAR(voiceLinesMorseCodeList), GVAR(voiceLinesMorseCodeWeights), "Morse Code"]];
 GVAR(voiceLinePacks) set ["electronic", 		[GVAR(voiceLinesEletronicList), GVAR(voiceLinesElectronicWeights), "Electronic"]];
 GVAR(voiceLinePacks) set ["alienElectronic", 	[GVAR(voiceLinesAlienEletronicList), GVAR(voiceLinesAlienElectronicWeights), "Alien Electronic"]];
-GVAR(voiceLinePacks) set ["police", 			[GVAR(voiceLinesPoliceList), GVAR(voiceLinesPoliceWeights), "Police Radio"]];
\ No newline at end of file
+GVAR(voiceLinePacks) set ["police", 			[GVAR(voiceLinesPoliceList), GVAR(voiceLinesPoliceWeights), "Police Radio"]];
+
+
+// Spectrum autoline settings
+GVAR(spectrumAutolineColours) = ["ColorBlack", "ColorGrey", "ColorRed", "ColorBrown", "ColorOrange", "ColorYellow", "ColorKhaki", "ColorGreen", "ColorBlue", "ColorPink", "ColorWhite", "ColorWEST", "ColorEAST", "ColorGUER", "ColorCIV"]; // Must only contain colours from CfgMarkerColors
```

Comment:
I was maybe thinking more of 
```sqf
GVAR(spectrumAutolineColours) = getArray ( configfile >> "CfgMarkerColors" >> _color >> "color" );
```
To dynamically get all the colors once as the game loads, and populate it. So if some mod adds more colors you have them available? 

Or does this mess with the default colors as you state below?

Reply:
It would (probably?) make the default colours inconsistent depending on whether you had custom colours or not, as they're accessed by index. It'd also (probably) mean leaving the labels as "ColorBlack" (e.g. however they're defined in CfgColors) as I don't think we could guarantee they'd all be named in the same format.

If you think it's worth it to include custom colours, am happy to make the change!

Reply:
That is a good point. As previous settings might change if you load mods that change the order of the Cfg. While I would assume it would always append to basegame array, I am not certain. Might as well leave it as it is. Plenty of options for colour anyway as it is.